utils/rpms/daos.spec

```diff
@@ -509,13 +509,16 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # set daos_firmware to be setuid root in order to perform privileged tasks
 %attr(4750,root,daos_server) %{_bindir}/daos_firmware
 
-%files daos_serialize
+%files serialize
 %{_libdir}/libdaos_serialize.so
 
 %files tests
 # No files in a meta-package
 
 %changelog
+* Mon Nov 03 2021 Danielle M. Sikich <danielle.sikich@intel.com> 2.1.100-6
```

Comment:
3rd November was a wednesday.

We get lots of these sneak in, and they cause warnings but not failures then every month somebody has to go through and correct them all.